specs/linux/http_events.table

```diff
@@ -0,0 +1,21 @@
+table_name("http_events")
+description("Tracks HTTP request header.")
+schema([
+    Column("time", BIGINT, "Time of HTTP event"),
+    Column("method", TEXT, "HTTP request method GET/POST/PUT/DELETE/OPTIONS/PATCH"),
+    Column("protocol", TEXT, "SSL protocol used in communication(only for HTTPS/TLS)"),
+    Column("local", TEXT, "IP address of the local interface"),
+    Column("remote", TEXT, "IP address of the HTTP responder"),
+    Column("s_port", BIGINT, "Source port"),
+    Column("d_port", BIGINT, "Destination port"),
+    Column("host", TEXT, "Host name available in HTTP header"),
+    Column("port", INTEGER, "Port number from HTTP URL"),
+    Column("uri", TEXT, "URI added to HTTP host name"),
+    Column("content_type", TEXT, "Content type of HTTP request"),
+    Column("user_agent", TEXT, "Client making HTTP request"),
+    Column("ja3", TEXT, "Parameters responsible for JA3 calculation(only for HTTPS/TLS)"),
```

Comment:
What is the format of this TEXT string? Naively it seems like it should be `ja3_raw` and hidden. And the hash should be `ja3`